include/range/v3/iterator_range.hpp

```diff
@@ -189,30 +192,6 @@ namespace ranges
             constexpr auto&& make_iterator_range = static_const<make_iterator_range_fn>::value;
         }
 
-        /// Tuple-like access for `iterator_range`
-        template<std::size_t N, typename I, typename S>
-        constexpr auto get(iterator_range<I, S> & p)
-        RANGES_DECLTYPE_AUTO_RETURN_NOEXCEPT
-        (
-            ranges::get<N>(static_cast<tagged_pair<tag::begin(I), tag::end(S)> &>(p))
-        )
-
-        /// \overload
-        template<std::size_t N, typename I, typename S>
-        constexpr auto get(iterator_range<I, S> const & p)
-        RANGES_DECLTYPE_AUTO_RETURN_NOEXCEPT
-        (
-            ranges::get<N>(static_cast<tagged_pair<tag::begin(I), tag::end(S)> &>(p))
-        )
-
-        /// \overload
-        template<std::size_t N, typename I, typename S>
-        constexpr auto get(iterator_range<I, S> && p)
-        RANGES_DECLTYPE_AUTO_RETURN_NOEXCEPT
-        (
-            ranges::get<N>(static_cast<tagged_pair<tag::begin(I), tag::end(S)> &&>(p))
-        )
-
```

Comment:
I feel like there was a reason why I added these instead of relying on the base-to-derived conversion. :-P